Sources/TensorFlow/Layers/Dropout.swift

```diff
@@ -73,3 +73,31 @@ public struct Dropout<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
     }
   }
 }
+
+public struct GaussianDropout<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
```

Comment:
Doc comment?

Reply:
Ah my bad. Sorry to have missed it. I'll add it as soon as possbile ?